docs/guide/index.md

```diff
@@ -95,26 +95,29 @@ vitest related /src/index.ts /src/hello-world.js
 
 ### CLI Options
 
-| Options       |               |
-| ------------- | ------------- |
-| `-v, --version` | Display version number |
-| `-r, --root <path>` | Define the project root |
-| `-c, --config <path>` | Path to config file |
-| `-u, --update` | Update snapshots |
-| `-w, --watch` | Watch mode |
-| `-o, --open` | Open UI (default: false) |
-| `-t, --testNamePattern <pattern>` | Run tests with names matching the pattern |
-| `--api` | Listen to port and serve API |
-| `--threads` | Enable Threads (default: true) |
-| `--silent` | Silent console output from tests |
-| `--reporter <name>` | Select reporter: `default`, `verbose`, or `dot` |
-| `--coverage` | Use c8 for coverage |
-| `--run` | Do not watch |
-| `--global` | Inject APIs globally |
-| `--dom` | Mock browser api with happy-dom |
-| `--environment <env>` | Runner environment (default: node) |
-| `--passWithNoTests` | Pass when no tests found |
-| `-h, --help` | Display available CLI options |
+| Options                           |                                                                                                                                          |
```

Comment:
I dont think it is a good idea to format the table in this way, if we add a longer option the whole table will appear in the change set. What about having the option name and description in different lines?

Reply:
upps, sorry, IntelliJ doing its things, there is no lint rules for md files?

Reply:
or are you talking about chaning the format? I just copy/paste a few lines from existing ones

Reply:
ok, any preference or just like CLI Commands?

Reply:
Yeah, maybe you just revert to the way it was before

Reply:
@dammy001 I cannot find any commit with old options, directly included via table on #354

Reply:
I'll change it

Reply:
removed table, I include the option with h4 heading, and for Serve API with h5 heading (this way all options will not be shown on the menu)

Reply:
maybe we can use a simple md list

Reply:
Sorry, I actually like the table as a quick reference here. Could you revert for this PR, and in any case we can discuss this later? If it is because of my comment about different lines, it was just about table formatting and only as an option to what you sent before

Reply:
Could you revert the formatting of the table, so the changeset only adds the new option?

Reply:
Upps, I have lint on save... it could be hard, I Will change it externally with another ide

Reply:
Or modify and "Save without Formatting", I'm sure it is an option in IntelliJ
We should add linting to the docs

Reply:
upps, missing ending pipe for new api entries, will commit later...